spec/helpers/retina_image_helper_spec.rb

```diff
@@ -5,7 +5,7 @@
 
   before { allow(helper).to receive(:image_path) { |file| "/images/#{file}" } }
 
-  it { assert_match(%r{\A<img src=".*" srcset=".*" alt=".*" />\z}, tag) }
+  it { assert_match(%r{\A<img src=".*" srcset=".*" alt=".*"\s*/?>}, tag) }
```

Comment:
Not sure what happened here but it seems that sometimes the Rails tag helper ends with ` />` and sometimes it's `>`. I stopped caring and just made the regex support both